Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -0,0 +1,39 @@
+using Doppler.BillingUser.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.BillingUser.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class BillingController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/currentbillinginformation")]
+        public string GetCurrentBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to GetCurrentBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPut("/accounts/{accountname}/billinginformation")]
+        public string UpdateBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to UpdateBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/current-payment-method")]
+        public string GetCurrentPaymentMethod(string accountname)
+        {
+            return $"Hello! \"you\" that have access to GetCurrentPaymentMethod with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPost("/accounts/{accountname}/billing")]
+        public string TryBilling(string accountname)
```

Comment:
What does this URL represent?

Is it a bill, an invoice, an upgrade a checkout?

I mean, it would be nice to find a noun that represents this resource, for example:
```suggestion
        [HttpPost("/accounts/{accountname}/upgrade")]
        public string Upgrade(string accountname)
```

(I am not sure about _upgrade_)

Reply:
I didnt know how can I call that resource. I like "Upgrade".